### ISSUE TITLE
Ensure markup documents contain well-formed elements

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2538]
**OCIO Violation**: Ensure markup documents contain well-formed elements

**OCIO Notes:** 
This BUTTON has an id attribute of 'full-width-tab-2', which is not unique in the same DOM
This DIV has an id attribute of 'chart_div', which is not unique in the same DOM
This DIV has an id attribute of 'full-width-tabpanel-0', which is not unique in the same DOM
This path has an id attribute of 'Fill-6', which is not unique in the same DOM
This svg has an id attribute of 'Layer_1', which is not unique in the same DOM

**OCIO priority:** 1.9
More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/public/reporting/prioritization_tool.php?report_id=1532)

**Instances** (by note)

1. This BUTTON has an id attribute of 'full-width-tab-2', which is not unique in the same DOM

- [ ] https://revenuedata.doi.gov/how-revenue-works/    

2. This DIV has an id attribute of 'chart_div', which is not unique in the same DOM

- [ ] https://revenuedata.doi.gov/fact-sheet/ - bar charts     

3. This DIV has an id attribute of 'full-width-tabpanel-0', which is not unique in the same DOM

- [ ] https://revenuedata.doi.gov/how-revenue-works/#Ownership-and-governance  - tool tip    

4. This path has an id attribute of 'Fill-6', which is not unique in the same DOM

- [ ] https://revenuedata.doi.gov/how-revenue-works - two things on this page give this note. See spreadsheet for more info    

5. This svg has an id attribute of 'Layer_1', which is not unique in the same DOM

- [ ] https://revenuedata.doi.gov/query-data/?dataType=Federal%20revenue%20by%20company - Query Data & Download Icons

- [ ] https://revenuedata.doi.gov/ - Filter Table Icon

- [ ] https://revenuedata.doi.gov/query-data/?dataType=Disbursements – icons

- [ ] https://revenuedata.doi.gov/how-revenue-works/#how-are-natural-resources-on-native-american-land-governed – icons

- [ ] https://revenuedata.doi.gov/ - header logo

- [ ] https://revenuedata.doi.gov/how-revenue-works/#understanding-federal-revenues - Major commodities in the US ribbon graphic

**Comment by Maroyafaied** [comment]
Instance #5 might be resolved by issue #2524

**Comment by eelzi-ONRR** [comment]
2024 report

Instances: 846
Occurrence: 25%
Severity: 6
Priority: 1


Instance 5 listed in the description  was not solved by issue #2524. The most recent report still shows 278 instances of that cause for the violation

**Comment by lpgoldstein** [comment]
No longer relevant
